### DISCUSSION TITLE
PIXI.Graphics vertices resolution

### DISCUSSION THREAD

**Discussion by jbjanot** [pixijs/pixijs#8373] [Q&A] [answered]
When using the PIXI.Graphics class to draw shapes, I discovered I could get a huge performance boost by drawing my shapes only a fraction of the wanted size and then scale it back to fullscreen, because of the reduction of the number of vertices used to draw the shapes.

I wonder if there is a more direct way to tell PIXI to draw shapes with a reduced resolution/with less vertices? I could not find anything in the docs.

Any idea on how to achieve this? 

Thank you!

**Comment by bigtimebuddy** [comment, marked as answer]
You may want to checkout [GRAPHICS_CURVES](http://pixijs.download/dev/docs/PIXI.html#GRAPHICS_CURVES) options.  You can try increasing the `maxSegments` and `maxLength`.

**Reply by jbjanot** [reply]
~~Thank you @bigtimebuddy for your answer, as it seems indeed what I am looking for. Could you please help me to understand in which way I can take advantage of this property when drawing shapes ? Is it a constant that I need to edit before starting to draw, or a parameter to add when creating the Graphics object ?~~
Nevermind, I just need to edit the properties before drawing, that's all. Thanks a lot for your help!
